content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+service/nimble-csp-svc        ClusterIP   10.102.45.17     <none>        8080/TCP   69d
+service/primera3par-csp-svc   ClusterIP   10.104.57.79     <none>        8080/TCP   69d
+
+NAME                          DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/hpe-csi-node   3         3         3       3            3           <none>          69d
+
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/hpe-csi-controller    1/1     1            1           69d
+deployment.apps/nimble-csp            1/1     1            1           69d
+deployment.apps/primera3par-csp       1/1     1            1           69d
+deployment.apps/snapshot-controller   2/2     2            2           32d
+
+NAME                                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/hpe-csi-controller-7c6f876494    1         1         1       69d
+replicaset.apps/nimble-csp-db7c7bb65             1         1         1       69d
+replicaset.apps/primera3par-csp-6f999b8d76       1         1         1       69d
+replicaset.apps/snapshot-controller-64b98b668f   2         2         2       32d
+
+NAME                                   READY   AGE
+statefulset.apps/snapshot-controller   1/1     69d
+```
+
+### Step-5: Create a namespace on 'hpe' Kubernetes cluster for MongoDB Deployment
+
+```shellsession
+$ kubectl create ns mongo
+namespace/mongo created
+
+$ kubectl get ns
+NAME                      STATUS   AGE
+argocd                    Active   71d
+default                   Active   71d
+ezctl                     Active   71d
+gatekeeper-system         Active   71d
+hpe                       Active   71d
+hpe-externalclusterinfo   Active   71d
+hpe-ldap                  Active   71d
+hpe-logzio                Active   71d
+hpe-metering              Active   71d
+hpe-nodesvc               Active   71d
+hpe-operations            Active   50d
+hpe-secure                Active   71d
+hpe-security              Active   71d
+hpe-snow                  Active   71d
+hpe-storage               Active   71d
+hpe-system                Active   71d
+hpe-templates-compute     Active   71d
+hpecp                     Active   71d
+hpecp-bootstrap           Active   71d
+hpecp-falco               Active   71d
+kd-apps                   Active   71d
+kd-mlops                  Active   71d
+kube-node-lease           Active   71d
+kube-public               Active   71d
+kube-system               Active   71d
+kubernetes-dashboard      Active   71d
+mongo                     Active   6s
+opsramp-agent             Active   71d
+
+Switch the new namespace in the current context
+
+$ kubectl config set-context --current --namespace=mongo
+Context "caas-dev-3-hpe-hpe-ashish-kumar@hpe.com" modified.
+
+$ kubectl config view --minify --output 'jsonpath={..namespace}'; echo
+mongo
+
+$ kubectl get pods
+No resources found in mongo namespace.
+```
+
+### Step-6: Deploy MongoDB
```

Comment:
Same comment here. Provide a little guidance/instruction/explanation.

Reply:
Here we are showing to end user, how to execute the cmd or deploy the application and showing the content of the file.
Looking forward for more inputs in this area.